fiboa_cli/datasets/dk.py

```diff
@@ -0,0 +1,163 @@
+
+# Description in also adapted from https://collections.eurodatacube.com/denmark-lpis/
+DESCRIPTION = """Danish Land Parcel Identification System (LPIS) data collection or (field blocks/ 'Markblokke') is managed by The Danish Agency for Agriculture.
+
+The field block map is a digital field map, with agricultural areas gathered in field blocks. A field block is a geographically coherent unit consisting of agricultural land. The boundaries of the field blocks typically follow permanent divisions in the landscape. The map is used in the administration of cases linked to the geographical determination of cultivation areas, primarily by the EU's area-based support schemes. The field block map is continuously updated on the basis of orthophotos and reports from farmers and authorities. The field block map covers approx. 2.8 million hectares of agricultural land. Each field block is identified with a field block number, area, type. A field block means a continuous area on which one or more farmers grow one or more different crops. The map is updated every night with changes from case processing."""
+# Bounding box of the data in WGS84 coordinates
+BBOX = [8.7378109937414692,10.4170362955184004, 55.1917137800704012, 57.4329738730357988]
+
+# Provider name, can be None if not applicable, must be provided if PROVIDER_URL is provided
+PROVIDER_NAME = "The Danish Agency for Agriculture"
+# URL to the homepage of the data or the provider, can be None if not applicable
+PROVIDER_URL = "https://geodata-info.dk/srv/eng/catalog.search#/metadata/d91b2c99-d9b0-4e6d-b323-20ac80548186"
+# Attribution, can be None if not applicable
+ATTRIBUTION = "The Danish Agency for Agriculture"
+
+# License of the data, either
+# 1. a SPDX license identifier (including "dl-de/by-2-0" / "dl-de/zero-2-0"), or
+LICENSE = "CC0-1.0"
+# 2. a STAC Link Object with relation type "license"
+# LICENSE = {"title": "CC-BY-4.0", "href": "https://creativecommons.org/licenses/by/4.0/", "type": "text/html", "rel": "license"}
+
+# Map original column names to fiboa property names
+# You also need to list any column that you may have added in the MIGRATION function (see below).
+COLUMNS = {    
+    'MARKBLOKNR' : 'id', # fiboa core field , TODO MARKBLOKNR-id conversion is needed.     
+    'GEOMETRISK' : 'area',#fiboa custom field i.e total area = eligible area + ineligible area
+    'TARAAREAL': 'ineligible_area', #fiboa custom field
+    'STOETPROC' : 'support_proc',  # fiboa custom field - translated. Supported Scheme. Currently all values are NaN
+    'GB_FRADRAG' : 'deduction', # fiboa custom field - translated 
+    'MB_TYPE' : 'block_type', #fiboa custom field
+    'GB_AREAL': 'eligible_area', # fiboa core field i.e. eligible area
+    'geometry':'geometry'# fiboa core field
+}
+
+# Add columns with constant values.
+# The key is the column name, the value is a constant value that's used for all rows.
+ADD_COLUMNS = {
+    "determination_datetime": "2021-01-01T00:00:00Z"
```

Comment:
Is this a leftover from the template?

Reply:
No, so the dataset doesn't have any date attributes for individual parcels but from the metadata, it is possible to identify the year in which the data was collected. So for simplicity, we thought we could choose the template date 01.01.XXXX .

Because in the actual fieldscapes dataset, we have the harvest and plantation season's data from the seasonal crop calendar for the given year.